lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -62,6 +62,11 @@ class EnrollAlertSerializer(serializers.Serializer):
     extra_text = serializers.CharField()
 
 
+class ResumeCourseSerializer(serializers.Serializer):
+    has_visited_course = serializers.BooleanField()
+    url = serializers.CharField()
```

Comment:
Should this be a UrlField? I think it might just add in some validation, but potentially nice?